### PR TITLE
Develop to programs merge 30 03 2023

### PR DESCRIPTION
**Pull request by clemens-msupply**
Please check that if `yarn run generate` still works because there where some `sed` linux/mac issues that have been fixed...

Best to look at the commits after the main merge (nothing that stands out though)

### CODE REVIEW

File: client/packages/programs/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "tsc": "tsc"
   },
   "dependencies": {
-    "@mui/material": "^5.2.3",
+    "@mui/material": "^5.11.13",
```
[the remark this answers is not in the document]
Comment (mark-prins) [reply client/packages/programs/package.json:10]: best not to. Partly to limit the bundle size and partly to manage dependencies.  The general approach has been to import external dependencies only into common and re-export (android and electron being exceptions!)

**Comment by mark-prins** [comment]
I've added commit 6b9546741 to update some imports : importing directly from 'packages/' will break the jest tests. 
Now there's only four tests failing - and those are already fixed in develop or open PRs for develop.

File: client/packages/programs/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (clemens-msupply) [reply client/packages/programs/package.json]: thanks I "quickly" fixed them

**Review by mark-prins — APPROVED** [review]
😮  it's a lot of changes.. 

it was like a little trip down memory lane.. reviewing things I've seen over the last few weeks.
Have run tests, typescript validation, storybook, and run manual tests of the site. all looking good!

**Comment by clemens-msupply** [comment]
thanks Mark!

**Pull request merged**